src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,56 @@
+package duke.task;
+
+import duke.DukeException;
+import duke.util.DateTimeHandler;
+
+/**
+ * Represents an event as a task with a date and time.
+ */
+public class Event extends Task {
+
+    /** Date and time of the event. */
+    private final String dateTime;
+
+    /**
+     * Creates a new event from a description and a date and time String.
+     *
+     * @param description Description of the event.
+     * @param at Date and time of the event.
+     * @throws DukeException If the date and time input is invalid.
+     */
+    public Event(String description, String at) throws DukeException {
```

Comment:
Consider changing the variable at into something more descriptive.

Reply:
Changed from at to dateTime.